components/NavBar.tsx

```
@@ -134,33 +125,33 @@ export const NavBar = () => {
 
       {isMobileMenuOpen && (
         <div
-          className="fixed inset-0 bg-black bg-opacity-50 z-20 md:hidden"
+          className="bg-opacity-50 fixed inset-0 z-20 bg-black md:hidden"
           onClick={() => setIsMobileMenuOpen(false)}
         >
           <div
-            className="fixed top-16 left-0 right-0 bg-gray-400 shadow-lg"
+            className="fixed top-16 right-0 left-0 bg-gray-400 shadow-lg"
             onClick={(e) => e.stopPropagation()}
           >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)